src/main/java/se/sundsvall/casedata/service/scheduler/emailreader/EmailReaderWorker.java

```diff
@@ -47,7 +47,8 @@ public EmailReaderWorker(final MessageRepository repository, final ErrandReposit
 		this.notificationService = notificationService;
 	}
 
-	void getAndProcessEmails() {
+	@Transactional
+	public void getAndProcessEmails() {
```

Comment:
Conflicting changes with the ones in #158

Reply:
Yes, there were some leftovers from previous changes before christmas. Reverted now.